hack/make-rules/test-cmd.sh

```diff
@@ -66,6 +66,7 @@ function run_kube_apiserver() {
 
   "${KUBE_OUTPUT_HOSTBIN}/kube-apiserver" \
     --bind-address="127.0.0.1" \
+    --allow-privileged=true \
```

Comment:
I don't think, we have sufficient reasoning to this dramatic change. It is better to remove this.

Reply:
Along with fixing netadmin profile, I've fixed remove it.